packages/@aws-cdk/core/lib/removal-policy.ts

```diff
@@ -16,7 +16,8 @@ export interface RemovalPolicyOptions {
   /**
    * The default policy to apply in case the removal policy is not defined.
    *
-   * @default RemovalPolicy.Retain
+   * The default value is resource specific. To determine the default value for a resoure,
```

Comment:
Would prefer you kept this under the `@default - ` tag . Go `@default - Default value is resource specific ... `